gulpfile.babel.js

```diff
@@ -31,6 +31,14 @@ gulp.task( 'old-styles:watch', function() {
 	return gulp.watch( 'scss/**/*.scss', gulp.parallel( 'old-styles' ) );
 } );
 
+gulp.task( 'blocks:watch', function() {
+	const child = require( 'child_process' ).execFile( 'yarn', [ 'build-extensions', '--watch' ] );
```

Comment:
To get the most developer oriented builds, you can add the most verbose sourcemaps and set development `NODE_ENV`. The former is nice if you're ever inspecting the source and the latter is probably redundant:

```js
const child = require( 'child_process' ).execFile( 'yarn', [ 'build-extensions', '--watch' ], {
	env: {
		NODE_ENV: 'development',
		SOURCEMAP: 'source-map',
	},
} );
```